content/community/contributing/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to contribute"
+autotoc: false
+---
+<div class='right'><img src="/images/undraw-illustrations/contribute.svg" alt="" width="240" /></div>
+
+If you want to contribute to the Galaxy Project, there are plenty of ways to do it. And here is the good news: __you can contribute with code, but not only with code__!
```

Comment:
Maybe instead of **_you can contribute with code, but not only with code_**  change for   **_there are many ways to do so, not only with code!_**